Failing while trying to compile android release

# Bug Report

**Before opening**

- [x] Did you try the latest release? 6.0.2-RC
- [x] Did you look for existing issues? Yes

**Ad Modules**
Banner Ads

**Platforms**
Android

**Versions**
- react-native-fbads: 6.0.2-RC
- react-native: 0.56.1

**Ads Environment**

`react-native run-android --variant=release` 
throws the following error:
```
:react-native-fbads:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/Users/danrevah/dev/private/xxxxxxxx-react-app/node_modules/react-native-fbads/android/app/src/main/java/suraj/tiwari/reactnativefbads/NativeAdManager.java:108: error: cannot find symbol
    String error = String.format("%d: %s", adError.getErrorCode(), adError.getErrorMessage(), Locale.ENGLISH);
                                                                                              ^
  symbol:   variable Locale
  location: class NativeAdManager
Note: /Users/danrevah/dev/private/xxxxxxxx-react-app/node_modules/react-native-fbads/android/app/src/main/java/suraj/tiwari/reactnativefbads/BannerViewManager.java uses unchecked or unsafe operations.
```

--- a/android/app/src/main/java/suraj/tiwari/reactnativefbads/NativeAdManager.java
+++ b/android/app/src/main/java/suraj/tiwari/reactnativefbads/NativeAdManager.java
@@ -25,6 +25,7 @@ import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Locale;
 
 public class NativeAdManager extends ReactContextBaseJavaModule implements NativeAdsManager.Listener {
   /**

--- a/ios/ReactNativeAdsFacebook.xcodeproj/project.pbxproj
+++ b/ios/ReactNativeAdsFacebook.xcodeproj/project.pbxproj
@@ -305,7 +305,7 @@
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PROJECT_DIR)/**",
-					"$(SRCROOT)/../../../ios/Pods/FBAudienceNetwork",
+					"$(SRCROOT)/../../../ios/Pods/FBAudienceNetwork/**",
 				);
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
@@ -324,7 +324,7 @@
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PROJECT_DIR)/**",
-					"$(SRCROOT)/../../../ios/Pods/FBAudienceNetwork",
+					"$(SRCROOT)/../../../ios/Pods/FBAudienceNetwork/**",
 				);
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",

